### PR TITLE
WL-3318

### PR DESCRIPTION
**Pull request by nicholaswilson100**
Remove irrelevant bits of tutorial & use English spellings

d/ change to UK spellings, eg, _organise_ not organize
e/ as step 5 (or 4 or 6) of the tutorial says "Profile Tool" - I cant see where it is pointing but I don’t think you can access your Profile, you can however access your "Preferences" so change it to Preferences Tool
f/ change "My Workspace" to My Dashboard" throughout the tutorial

### CODE REVIEW

**Comment by buckett** [comment]
All is ok. Only comments are:
- Profile Tool / Preferences, this should go back to Sakai so would be easier todo if was in it's own commit.
- Sakai -> WebLearn would be nice to have this using a positional parameter in the i18n strings, but not sure how worthwhile this is.
- We probably only need to change en_GB for the Oxford only changes as everyone is forced to use this locale.

**Comment by nicholaswilson100** [comment]
The tutorial is mainly in javascript (/reference/library/src/webapp/js/jquery/qtip/tutorial.js) so I'm kind of put off trying to use positional parameters as I'm don't really know how to do that from JS (as opposed to Java)...

**Comment by buckett** [comment]
I think tutorial.js uses AJAX to load the actual translated strings, these are built in org.sakaiproject.component.app.help.TutorialEntityProviderImpl

**Comment by nicholaswilson100** [comment]
Great thanks :)

**Comment by nicholaswilson100** [comment]
Added positional parameter for 'WebLearn'

File: help-component/src/java/org/sakaiproject/component/app/help/TutorialEntityProviderImpl.java [hunk 1]
```diff
@@ -83,8 +84,8 @@ public Object getEntity(EntityReference ref) {
 		valuesMap.put("nextUrl", nextUrl);
 		
 		//build the body html:
-		String body = msgs.getString(ref.getId() + ".body");
-		
+		String body = replacePosParam(msgs.getString(ref.getId() + ".body"), "siteName");
```
Comment (buckett) [inline help-component/src/java/org/sakaiproject/component/app/help/TutorialEntityProviderImpl.java]: siteName can come from ServerConfigurationService with the ui.service key as we did in the calendar iCal issue.

**Comment by nicholaswilson100** [comment]
Make site.name come from sakai.properties

File: help-component/src/java/org/sakaiproject/component/app/help/TutorialEntityProviderImpl.java [hunk 2]
```diff
@@ -83,8 +86,9 @@ public Object getEntity(EntityReference ref) {
 		valuesMap.put("nextUrl", nextUrl);
 		
 		//build the body html:
-		String body = msgs.getString(ref.getId() + ".body");
-		
+		String siteName = serverConfigurationService.getString(SITE_NAME, "WebLearn");
+		String body = replacePosParam(msgs.getString(ref.getId() + ".body"), siteName);
```
Comment (buckett) [inline help-component/src/java/org/sakaiproject/component/app/help/TutorialEntityProviderImpl.java]: I think you can just use `msg.getFormattedMessage(String, Object[])`

**Comment by nicholaswilson100** [comment]
Thanks. Made the change.

File: help-component/src/java/org/sakaiproject/component/app/help/TutorialEntityProviderImpl.java [hunk 3]
```diff
@@ -18,13 +14,21 @@
...
+import java.net.URL;
+import java.text.MessageFormat;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 
 public class TutorialEntityProviderImpl implements TutorialEntityProvider, AutoRegisterEntityProvider, RESTful{
 
+	public static final String SITE_NAME = "site.name";
+	public static final String WEB_LEARN = "WebLearn";
```
Comment (buckett) [inline help-component/src/java/org/sakaiproject/component/app/help/TutorialEntityProviderImpl.java]: Sorry I didn't spot this before, the default should be Sakai so we can push this back to Sakai.

**Comment by nicholaswilson100** [comment]
Oh yes should have realised - changed "WebLearn" to "Sakai"

**Pull request merged**